.env.example

```diff
@@ -2,3 +2,9 @@ PORT=3333
 HOST=0.0.0.0
 NODE_ENV=development
 APP_KEY=0ttwaajIw4mEJqZx3xLyGjqN1Lj_ur7y
```

Comment:
What is app key :D

Reply:
The APP_KEY is a random 32-character string stored in the APP_KEY in your projects **.env** file. 
It is used for encrypting cookies and creating the signature for signed URLs and queued closures.

Reply:
Encryption of cookies, signes URLs and queued closures, is this Adonis out of the box stuff? Or how would you use it?

Reply:
This is Adonis out-of-the-box stuff, however, the app_key is a private string that nobody knows about, so if only your application knows the app_key, only your application can decrypt the data that is encrypted by this key, which is a security advantage and that is why it is stored in a **.env** file that is not tracked by version control and if not configured in a **.env**, all sessions and other encrypted data will not be secure.

Reply:
Okok, the offer encryption features out of the box. I see. But you did check in your app key :P I guess that was a mistake? @arnoldkunihira